modules/websocket/websocket_server.h

```diff
@@ -48,6 +48,7 @@ class WebSocketServer : public WebSocketMultiplayerPeer {
 	Ref<CryptoKey> private_key;
 	Ref<X509Certificate> ssl_cert;
 	Ref<X509Certificate> ca_chain;
+	uint32_t handshake_timeout = 3000;
```

Comment:
Is it intended that it's `uint32_t` here but `uint64_t` in `do_handshake()`?

Reply:
Well, 32 bits unsigned is enough for the timeout, and `do_handshake` matches the type of time functions, I can make that a `uin32_t`  too, but making the `handshake_timeout` 64 bits is seems like a waste.